agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/InheritedAttributesLogProcessor.java

```diff
@@ -71,6 +71,7 @@ private static List<AttributeKey<?>> buildInheritedAttributesList(
   public void emit(LogData log) {
     Span currentSpan = Span.current();
     if (!(currentSpan instanceof ReadableSpan)) {
+      delegate.emit(log);
```

Comment:
this is considered a log not part of the trace?

Reply:
yes, we capture logs that are not part of a request also